packages/react-ui/components/FxInput/__stories__/FxInput.stories.tsx

```diff
@@ -3,14 +3,24 @@ import { CSFStory } from 'creevey';
 
 import { BGRuler } from '../../../internal/BGRuler';
 import { FxInput } from '../FxInput';
+import { Gapped } from '../../Gapped';
 import { createPropsGetter } from '../../../lib/createPropsGetter';
-import { InputType } from '../../Input';
+import { InputSize, InputType } from '../../Input';
 
 export default { title: 'FxInput' };
 
 export const TypeText = () => <TestFxInput />;
 TypeText.story = { name: 'type text' };
 
+export const DifferentSizesFxInput = () => (
+  <Gapped vertical>
+    <TestFxInput />
```

Comment:
Давай здесь укажем размер, на случай если мы решим изменить дефолтный размер.
```suggestion
    <TestFxInput size="small" />
```